src/page/utkast/footer/dialog-innhold.tsx

```diff
@@ -134,53 +134,41 @@ export function DialogInnhold(props: DialogFooterInnholdProps) {
 			<Show if={showSection}>
 				<div className="utkast-footer--innhold-sidestilt">
 					<Show if={visStartBeslutterProsess}>
```

Comment:
Glemt å fjerne bruk av `Show`-komponenten i denne filen? (ref at det er fjernet i andre filer til fordel for `&&` conditional-rendering operator)

Reply:
Jepp! Fikser 👷